pkgs/development/libraries/arrow-cpp/default.nix

```diff
@@ -31,22 +31,23 @@ in stdenv.mkDerivation rec {
   };
   sourceRoot = "apache-arrow-${version}/cpp";
 
-  ARROW_JEMALLOC_URL = fetchurl {
+  ARROW_JEMALLOC_URL = jemalloc.src;
+
+  ARROW_MIMALLOC_URL = fetchFromGitHub {
     # From
     # ./cpp/cmake_modules/ThirdpartyToolchain.cmake
     # ./cpp/thirdparty/versions.txt
-    url =
-      "https://github.com/jemalloc/jemalloc/releases/download/5.2.1/jemalloc-5.2.1.tar.bz2";
-    hash = "sha256-NDMOXOJ2CZ4uiVDZM121qHVomkxqVnUe87HYxTf4h/Y=";
+    owner = "microsoft";
+    repo = "mimalloc";
+    rev = "v1.7.2";
+    hash = "sha256-yHupYFgC8mJuLUSpuEAfwF7l6Ue4EiuO1Q4qN4T6wWc=";
   };
 
-  ARROW_MIMALLOC_URL = fetchurl {
-    # From
-    # ./cpp/cmake_modules/ThirdpartyToolchain.cmake
-    # ./cpp/thirdparty/versions.txt
-    url =
-      "https://github.com/microsoft/mimalloc/archive/v1.7.2.tar.gz";
-    hash = "sha256-sZEuNUVlpLaYQQ91g8D4OTSm27Ot5Uq33csVaTIJNr0=";
+  ARROW_XSIMD_URL = fetchFromGitHub {
```

Comment:
Does this gold a path to the nix store or the content itself?

Reply:
I assumed it was the nix store. Can derivations point to some other location?